mcs/class/corlib/Test/System/TimeZoneInfoTest.cs

```diff
@@ -786,7 +841,6 @@ public void TestAthensDST_InDSTDelta ()
 				Assert.IsTrue (tzi.IsDaylightSavingTime (new DateTimeOffset (date, tzi.GetUtcOffset (date))));
 			}
 
-			[Test]
```

Comment:
was this intentional?

Reply:
No that was unintentional - restored the `[Test]` attribute.